Add/update docs for new 1.15 features

khorne3 opened the pull request.

vapurrmaid reviewed (changes requested):
## Requested Changes

- Cluster Info screenshot shows "Edit", but that's internal-only right now as the feature is under development. I'd disable the feature flag and re-take the shot

- PWA page will need a further overhaul. We have removed the _root_ PWA (ie the majority of the dashboard), opting instead for a PWA-per-application approach.

khorne3 commented:
@vapurrmaid 

- Updated Cluster Info screenshot to exclude the edit button
- Changed the organization of the PWA doc so that legacy info is still available for those on older versions of Coder

vapurrmaid commented on environments/pwa.md:
```diff
@@ -1,25 +1,33 @@
 title: "Progressive Web Apps"
-description: Learn how to install the editor PWAs for a browser editing experience closer to native.
+description: Learn how to install the editor PWAs
+for a browser editing experience closer to native.
```
```suggestion
for a browser editing experience closer to native IDEs.
```

vapurrmaid commented on environments/pwa.md:
```diff
@@ -1,25 +1,33 @@
-a progressive web application (PWA). Using Coder as a PWA offers you
-an experience that is akin to a native application and offers improved
-performance.
+In addition to running Coder in a web browser, you can also run it as a
```
```suggestion
In addition to running Coder in a web browser, you can also run each application as a
```

Probably will fail lint as-is due to line-length, but just to demonstrate wording

vapurrmaid commented on environments/pwa.md:
```diff
@@ -1,25 +1,33 @@
+## Installing the PWA
+
+1. Log into Coder and select your environment.
+2. Under **Applications** click **code** to launch code-server.
```
```suggestion
2. Under **Applications**, click on an application in order to launch it as a new window (or tab)
```

vapurrmaid commented on environments/pwa.md:
```diff
@@ -1,25 +1,33 @@
+
+1. Log into Coder and select your environment.
+2. Under **Applications** click **code** to launch code-server.
+3. Follow your browser's instructions for installing the PWA.
```
```suggestion
3. Follow your browser's instructions for installing the application as a PWA.
```

vapurrmaid commented on environments/pwa.md:
```diff
@@ -1,25 +1,33 @@
+2. Under **Applications** click **code** to launch code-server.
+3. Follow your browser's instructions for installing the PWA.
+
+Please note that the code-server app is installed on a per-environment basis.
```
```suggestion
Please note that applications are currently installed on a per-environment basis.
```
